x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/ml/datafeed_high_count_by_user_agent_jsbase.json

```diff
@@ -0,0 +1,15 @@
+{
+  "job_id": "JOB_ID",
+  "indices": [
+    "INDEX_PATTERN_NAME"
+  ],
+  "max_empty_searches": 10,
+  "query": {
+    "bool": {
+      "must": [
+        { "bool": { "filter": { "term": { "agent.name": "js-base" } } } },
+        { "bool": { "filter": [{ "exists": { "field": "user_agent.name" } }] } }
```

Comment:
please add an additional filter for transaction to the must:

```
{"bool": {"filter": {"term": {"processor.event": "transaction"}}}}
```

Reply:
https://github.com/elastic/kibana/pull/53792/commits/2d1fe5c7e1d50d790dfd5bdc1607ec37c5279c9a